libs/sdk-ui-dashboard/src/presentation/widget/insight/DefaultDashboardInsight/InsightDrillDialog/InsightDrillDialog.tsx

```diff
@@ -124,22 +94,12 @@ export const InsightDrillDialog = (props: InsightDrillDialogProps): JSX.Element
                     onExportCSV={onExportCSV}
                     isLoading={isLoading}
                 >
-                    <DefaultDashboardInsightWithDrillSelect
-                        insight={insight}
-                        widget={widget}
-                        disableWidgetImplicitDrills
-                        onDrill={onDrill}
-                        onDrillDown={onDrillDown}
-                        onDrillToAttributeUrl={onDrillToAttributeUrl}
-                        onDrillToCustomUrl={onDrillToCustomUrl}
-                        onDrillToDashboard={onDrillToDashboard}
-                        onDrillToInsight={onDrillToInsight}
-                        drillTargets={drillTargets}
-                        onAvailableDrillTargetsReceived={onAvailableDrillTargetsReceived}
-                        onError={setError}
-                        onLoadingChanged={handleLoadingChanged}
-                        onExportReady={handleExportReady}
-                    />
+                    <WithDrillSelect insight={props.insight} onDrillDownSuccess={onDrillDown}>
+                        {({ onDrill }) => {
+                            return <DrillDialogInsight {...props} onDrill={onDrill}      onLoadingChanged={handleLoadingChanged}
```

Comment:
The formatting here is sketchy (extra spaces), maybe this is what is making the tests fail?